build/nuke/docker/alpine.dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0-alpine3.16
+
+RUN apk update \
```

Comment:
I suppose that we should pin exact versions here.

Reply:
I'll create security task & discussion.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/1137